less/svg.less

```diff
@@ -30,10 +30,13 @@
   }
 
   &.is-svg-paused .svg__playpause {
-    opacity: 0;
```

Comment:
I believe this was as such so that SR users could tab to it.

Reply:
https://github.com/cgkineo/adapt-svg/blob/25546c61495cf6fb42e3afa49669268366aa5a17/less/svg.less#L32

In the current Less on master, it is initially set to `opacity: 0` before immediately being overridden as `opacity: 1` in the next ruleset. So, I just removed the initial style which never actually gets used.

Starting on line 32:
```
  &.is-svg-paused .svg__playpause {
    opacity: 0;
    .icon-video-play;
  }

  html:not(.touch) &.is-svg-playing .svg__player:hover .svg__playpause,
  &.is-svg-paused .svg__playpause {
    opacity: 1;
    transition: none;
  }
```